app.js

```diff
@@ -0,0 +1,20 @@
+const navLinks = document.querySelector('.nav-links');
+const openNav = document.querySelector('.open-menu');
+const closeNav = document.querySelector('.close-menu');
+
+openNav.addEventListener('click', show);
+closeNav.addEventListener('click', close);
+
+/**  adding css style to the elements
+ *   for opening
+ */
+function show() {
+  navLinks.style.display = 'flex';
```

Comment:
- [ ] If you toggle between flex and no flex, the expected behavior will break, this happens when you open and close the hamburger, please fix it, you can use `display: none`
![image](https://user-images.githubusercontent.com/60631456/122836060-236b1300-d2b7-11eb-85c1-dc0067bff76b.png)